test/integration/api/uploader/uploader_spec.js

```diff
@@ -794,6 +794,43 @@ describe("uploader", function () {
     });
   });
 
+  it("should reject with promise rejection if disable_promises: false", function (done) {
+    const spy = sinon.spy();
+
+    cloudinary.v2.uploader.upload_large(EMPTY_IMAGE, { disable_promises: false }, () => {});
```

Comment:
this is also the default, right?

maybe we should explicitly test it without passing anything (to ensure the default value is picked up) - what do you think?

Reply:
added another test